R/chrome.R

```diff
@@ -118,8 +119,20 @@ chrome_print = function(
 
   box_model = match.arg(box_model)
 
+  is_shiny <- !is.null(shiny::getDefaultReactiveDomain())
```

Comment:
```suggestion
  is_shiny <- xfun::loadable('shiny') && !is.null(shiny::getDefaultReactiveDomain())
```

Reply:
as @cderv suggested, an `async` argument is added to `chrome_print`: shiny dependency is removed, and promises in now in **Suggests**